resize/resize3.c

```diff
@@ -105,18 +104,14 @@ int main(int argc, char *argv[])
     // write outfile's BITMAPINFOHEADER
     fwrite(&bi, sizeof(BITMAPINFOHEADER), 1, outptr);
     
-    // each pixel is 3 bytes, scanlineWidth is in bytes. (36 bytes -> 12 pixels);
-    // OG scanlineWidth
-    int oldRowWidth = og_biWidth*sizeof(RGBTRIPLE) + og_padding;
-    
     // allocate memory to temporarily store old image
     BYTE* old = calloc(og_biSizeImage, 1);
     
-    // new image's row width including padding in BYTES
-    int newRowWidth = bi.biWidth*sizeof(RGBTRIPLE) + padding; // in bytes
-
     //allocate memory to temporarily store new image
     BYTE* new = calloc(bi.biSizeImage, 1);
+    
+    // calculate new image's row width including padding in BYTES <-- TODO need this in pixels
```

Comment:
You can't have this in pixels. It is not guaranteed to be a multiple of `sizeof(RGBTRIPLE)`, that's the entire problem.